README.rst

```diff
@@ -228,16 +228,21 @@ Compatibility with GMT/Python/NumPy versions
       - GMT
       - Python
       - Numpy
+    * - `v0.5.0 <https://github.com/GenericMappingTools/pygmt/milestone/8>`_ (upcoming release)
+      - `dev Documentation <https://www.pygmt.org/dev>`_
```

Comment:
```suggestion
      - `Dev Documentation <https://www.pygmt.org/dev>`_ (reflects `main branch <https://github.com/GenericMappingTools/pygmt>`_)
```
We could do this instead of having the sentence below (haven't checked how it looks though).

Reply:
It will look like the screenshot below. Seems fine to me.

![image](https://user-images.githubusercontent.com/23487320/128947366-649286f6-faa8-4836-979c-3de05abf7b67.png)

Reply:
Looks good to me.

Reply:
Great. I think with this addition it is fine to remove the full sentence below.

Reply:
Done in 462d6a9648d106a7b4f0ac44bdabb52986d03dd4.